public/javascripts/SVLabel/src/SVLabel/navigation/MapService.js

```diff
@@ -1967,6 +1981,7 @@ function MapService (canvas, neighborhoodModel, uiMap, params) {
     self.timeoutWalking = timeoutWalking;
     self.resetWalking = resetWalking;
     self.getMoveDelay = getMoveDelay;
+    this.makeLinksClickable = makeLinksClickable;
```

Comment:
And if it turns out that you don't need to add that extra call to `makeLinksClickable()`, then you don't need to make it publicly available here either.